pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -545,6 +545,9 @@ type BareMetalHostStatus struct {
 	// OperationHistory holds information about operations performed
 	// on this host.
 	OperationHistory OperationHistory `json:"operationHistory"`
+
+	// ErrorCount records how many times the host has encoutered an error
```

Comment:
```suggestion
	// ErrorCount records how many times the host has encountered an error since the last successful operation
```

I think we want to make clear that the counter is reset between operations, so I've tried to update the wording to reflect that.